zlibWrapper/zstd_zlibwrapper.c

```diff
@@ -14,50 +14,48 @@
 #include "zstd_zlibwrapper.h"
 #define ZSTD_STATIC_LINKING_ONLY   /* ZSTD_MAGICNUMBER */
 #include "zstd.h"
-#define ZBUFF_STATIC_LINKING_ONLY  /* ZBUFF_createCCtx_advanced */
-#include "zbuff.h"
 #include "zstd_internal.h"         /* defaultCustomMem */
 
 
 #define Z_INFLATE_SYNC              8
-#define ZWRAP_HEADERSIZE            4
+#define ZLIB_HEADERSIZE             4
+#define ZSTD_HEADERSIZE             ZSTD_frameHeaderSize_min
 #define ZWRAP_DEFAULT_CLEVEL        5   /* Z_DEFAULT_COMPRESSION is translated to ZWRAP_DEFAULT_CLEVEL for zstd */
```

Comment:
It works like this:
- the compression level 0 for zlib is translated into Z_DEFAULT_COMPRESSION
- the compression level 0 for zstd is translated into ZWRAP_DEFAULT_CLEVEL
Since we are using zlibWrapper people might expect compression level 0 to get similar results from zstd as from zlib. I didn't make comparison but some time ago you've written that level 5 of zstd is similar to 6 of zlib.